build.cake

```diff
@@ -73,12 +84,21 @@ Task("ReplaceStubWithExample")
 Task("TestUsingExampleImplementation")
     .IsDependentOn("ReplaceStubWithExample")
     .Does(() => {
-        var allProjects = GetFiles(buildDir + "/*/*.csproj");
+        var allProjects = GetFiles(exercisesBuildDir + "/*/*.csproj");
         Parallel.ForEach(allProjects, parallelOptions, (project) => DotNetCoreTest(project.FullPath));
     });
 
+Task("BuildGeneratorsSolution")
+    .IsDependentOn("CopyGenerators")
+    .Does(() => {
+        var generatorsProject = GetFiles(generatorsBuildDir + "/*.csproj");
+        Parallel.ForEach(generatorsProject, parallelOptions, (project) => NuGetRestore(project.FullPath));
```

Comment:
We shouldn't need to call NuGetResore() explicitly since DotNetCoreBuild() will do an implicit restore for us.